tests/integration/explain.go

```diff
@@ -312,24 +313,41 @@ func findTargetNode(
 			}
 		}
 
+	case []any:
+		return findTargetNodeFromArray[any](targetName, toSkip, includeChildNodes, r)
+
```

Comment:
praise+question: interesting and this sorted out what you needed?

Reply:
Yeah we lose type info when serializing to and from JSON. This was the cleanest solution I could find for handling generic arrays.